inference-engine/tests/functional/shared_test_classes/src/base/ov_subgraph.cpp

```diff
@@ -26,6 +26,24 @@
 namespace ov {
 namespace test {
 
+std::ostream& operator <<(std::ostream& os, const std::vector<ov::Shape>& shapes) {
+    os << "{";
+    for (auto i = 0; i < shapes.size(); ++i) {
+        const auto& shape = shapes[i];
+        os << shape;
+        if (i < (shapes.size() - 1ul)) {
+            os << ", ";
```

Comment:
Please use "." instead of comma ",", some IDEs will skip all information after the "," sign. Moreover, yet another algorithm of dumping is used already (please see Eltwise and Softmax for the reference):

```cpp
    os << "(";
    auto itr = shapes.begin();
    do {
        os << CommonTestUtils::vec2str(*itr);
    } while (++itr != shapes.end() && result << "_");
    os << ")_";
```

Something like that, the main difference is using parentheses and dots.

Reply:
done